app/services/serial_normalizer.rb

```diff
@@ -3,11 +3,18 @@ def self.unknown_and_absent_corrected(str = nil)
     str = str.to_s.strip
     # Return unknown if blank, '?' or 'absent' (legacy concern - 'unknown' used to be stored as 'absent')
     return "unknown" if str.blank? || str.gsub(/\s|\?/, "").blank? || str.downcase == "absent"
-    return "made_without_serial" if str == "made_without_serial"
+    return "made_without_serial" if str == "made_without_serial" || looks_like_made_without?(str.downcase)
     return "unknown" if looks_like_unknown?(str.downcase)
     str
   end
 
+  def self.looks_like_made_without?(str_downcase)
+    case str_downcase
+    when /no serial/i then true
```

Comment:
I feel like the string "no serial" should look like unknown, rather than made_without_serial.

The bikes that are made without serial are few and far between, and generally custom. If the person is unable to correctly check the "made without serial" bike flow, it should be marked as unknown, rather than "made without"

Caveat: we should add the made without checkbox flow and functionality to the embed forms.

Reply:
Updated in 4488285c